tilings/tiling.py

```diff
@@ -1223,6 +1321,19 @@ def dimensions(self) -> Tuple[int, int]:
                 self._dimensions = (max(rows) + 1, max(cols) + 1)
         return self._dimensions
 
+    def underlying_tiling(self):
```

Comment:
Maybe `remove_assumption` would be a better name?

Reply:
sure